### DISCUSSION TITLE
Select form getInputProps

### DISCUSSION THREAD

**Discussion by jaallaall** [mantinedev/mantine#7164] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.12.2

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

None

### Describe the bug

Hello
form.getInputProps("name").onChange(value)
What is the use of
in Select component
Why when I use this method
form.getValues().name
is not updated
I sent an example so that you can understand why I use this method
Actually I have several components that are related to each other

https://codesandbox.io/p/sandbox/blissful-shadow-tfdph4?file=%2Fsrc%2FApp.js%3A29%2C11-29%2C52

### Self-service

- [X] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
Uncontrolled form mode does not trigger state updates, to watch for values changes, use `form.watch` hook or pass `onValuesChange` handler to `useForm` options. Alternatively, you can use controlled form mode and receive updated immediately in `form.values` object.
